full/src/test/java/apoc/load/LoadDirectoryTest.java

```diff
@@ -66,13 +86,455 @@ public void testWithNullUrlDir() {
         TestUtil.testFail(db, "CALL apoc.load.directory('*', null, {recursive: false}) YIELD value RETURN value", IllegalArgumentException.class);
     }
 
+    @After
+    public void clearDB() {
+        db.executeTransactionally("MATCH (n) DETACH DELETE n");
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void testAddListenerWithApocLoadDirectoryNotEnabled() {
+        apocConfig().setProperty(APOC_IMPORT_FILE_USE_NEO4J_CONFIG, true);
+        apocConfig().setProperty(APOC_LOAD_DIRECTORY_ENABLED, false);
```

Comment:
io aggiungerei che deve fallire se `apoc.import.file.enabled` è false, secondo te ha senso?

Reply:
Non so, abbiamo già messo `apoc.load.directory.enabled` come proprietà specifica per queste procedure, mettere anche `apoc.import.file.enabled` mi sembra qualcosa in più di superfluo. Tutt'al più, forse metterei `apoc.import.file.enabled` anche per questa e toglierei `apoc.load.directory.enabled`.

Reply:
@vga91 si fai così mi raccomando deve funzionare per db...